content/blog/2024-07-02-ctj-sponsorship.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Welcoming Cybertrust Japan again as a Platinum Sponsor Member"
+type: blog
+author: 
+ name: "benny Vasquez"
+ bio: "Chair, AlmaLinux OS Foundation"
+ image: /users/benny.jpeg
+date: '2024-07-02'
+images:
+  - /blog-images/2024/2024-07-cybertrust-sponsorship.png
+post: 
+    title: "Welcoming Cybertrust Japan again as a Platinum Sponsor Member"
+    image: /blog-images/2024/2024-07-cybertrust-sponsorship.png
+---
+
+Earlier today, we announced that our friends at [Cybertrust Japan Co., Ltd](https://www.cybertrust.co.jp/), have renewed their platinum sponsor membership. As a subsidiary of SB Technology Corp, this level of commitment to the foundation is huge, and we cannot thank them enough.
+
+Last year when Cybertrust Japan approached us, we knew that we were welcoming a partners. They are anticipating using AlmaLinux as the base for MIRACLE LINUX 10, and have spent the last year contributing to AlmaLinux in a number of ways, including expanding our SBOM support and they played a key role in organizing the first-ever [AlmaLinux Day: Tokyo](https://almalinux.org/aldt-2023/).
+
+As we highlighted in the press release, Cybertrust Japan's sizable contributions to AlmaLinux will allow us to expand AlmaLinux OS's security and adoption in Asia. We've also seen a noticeable upswing in the adoption of AlmaLinux in Japan!
+
+Since May of 2023...
+
+-   We have more than doubled the number of mirrors hosted in Japan
+-   We have nearly tripled the number of Japan-based ALOSF members
+-   We have grown the number of devices calling home for updates in Asia by 40%
+
+Cybertrust Japan's support is a clear catalyst for this, and we're so grateful!
+
+## What's Next?
+
+We plan to participate in the Open Source Summit in October, expand meetup group and user group participation, and participate in more open source conferences in Japan, as well as setting up another AlmaLinux Day: Tokyo this year!
```

Comment:
I like to include links to related things whenever I can, so everyone can easily catch up if they need to, maybe we can use:

```text
[Open Source Summit](https://events.linuxfoundation.org/open-source-summit-japan/)
```

I'm guessing this is the right link :smile:

Reply:
Good catch! I also like to include links. :)